src/main/java/it/gov/pagopa/pu/migration/dto/debtpositiontypeorgoperator/DebtPositionTypeOrgOperatorMigrationFileResult.java

```diff
@@ -0,0 +1,20 @@
+package it.gov.pagopa.pu.migration.dto.debtpositiontypeorgoperator;
+
+import it.gov.pagopa.pu.migration.wf.dto.MigrationFileResult;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import java.util.List;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+@NoArgsConstructor
+@AllArgsConstructor
+@SuperBuilder
+@ToString(callSuper = true)
+public class DebtPositionTypeOrgOperatorMigrationFileResult extends MigrationFileResult {
+  private Long organizationId;
+  private int numTotalRows;
+  private int numCorrectlyProcessedRows;
```

Comment:
do you plan to store these info on the db? upload or uploadDetail?

Reply:
ps: if the upload has many files, do you plan to store the sum or on the UploadDetail file-by-file? also the errorList: do you plan to store one file for entire input zip or one per unzipped file?